README.md

```diff
@@ -40,14 +40,16 @@ Usage
 
 `cibuildwheel` currently works on **Travis CI**, **Azure Pipelines**, **AppVeyor**, **GitHub Actions**, **CircleCI**, and **Gitlab CI**. Check the table below for supported platforms on each service:
 
-|                 | Linux | macOS | Windows |
-|-----------------|-------|-------|---------|
-| Azure Pipelines | ✅    | ✅    | ✅      |
-| Travis CI       | ✅    | ✅    | ✅      |
-| AppVeyor        | ✅    | ✅    | ✅      |
-| GitHub Actions  | ✅    | ✅    | ✅      |
-| CircleCI        | ✅    | ✅    |         |
-| Gitlab CI       | ✅    |       |         |
+|                 | Linux | macOS | Windows | Supports ARM |
+|-----------------|-------|-------|---------|--------------|
+| Azure Pipelines | ✅    | ✅    | ✅      | ✴️¹           |
+| Travis CI       | ✅    | ✅    | ✅      | ✅           |
+| AppVeyor        | ✅    | ✅    | ✅      |              |
+| GitHub Actions  | ✅    | ✅    | ✅      |              |
```

Comment:
also support self hosted ARM https://github.blog/changelog/2019-12-03-github-actions-self-hosted-runners-on-arm-architectures/

But We have no possibilities to test if cibuilwheel works properly on such machines.

Reply:
```suggestion
| GitHub Actions  | ✅    | ✅    | ✅      | ✴️¹           |
```

Reply:
Ahh, I just added this, assuming it would work, but maybe we should not if we can't test it? I guess Azure was tested at one point by someone? Though I really think it would work on GHA if it works on Azure, they are basically the same. And "third-party" already indicates a lot of work needs to be done to set it up.

Reply:
I suppose we could dial back the footnote to explain the situation: "These CIs require self-hosted runners for ARM. cibuildwheel probably works but this has not been confirmed."